### PR TITLE
[ASM][IAST] Update vulnerabilities list

### PR DESCRIPTION
**Pull request by e-n-0**
### What does this PR do? What is the motivation?

Update the list of possible detectable vulnerabilities with IAST (Code Security).

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by daniel-romano-DD — APPROVED** [review]
Thanks a lot

**Comment by e-n-0** [comment]
I don't understand why the filename is marked as invalid as I only edited the content of this file

File: content/en/security/application_security/code_security/_index.md [hunk 1]
```diff
@@ -23,17 +23,38 @@ Datadog is able to indicate the filename and line number where the vulnerability
...
+- Admin console active
 - Command Injection
-- Server Side Request Forgery (SSRF)
+- Default HTML escape invalid
+- Directory listing leak
+- Hardcoded Password
+- Hardcoded Secrets
+- Header Injection
+- HSTS Header missing
+- Insecure Auth Protocol
```
Comment (urseberry) [inline content/en/security/application_security/code_security/_index.md:34]:
```suggestion
- Hardcoded secrets
- Header injection
- HSTS header missing
- Insecure auth protocol
```

File: content/en/security/application_security/code_security/_index.md [hunk 2]
```diff
@@ -23,17 +23,38 @@ Datadog is able to indicate the filename and line number where the vulnerability
...
 - Insecure Cookie
+- Insecure JSP layout
+- LDAP injection
+- MongoDB Injection
```
Comment (urseberry) [inline content/en/security/application_security/code_security/_index.md:38]:
```suggestion
- MongoDB injection
```

File: content/en/security/application_security/code_security/_index.md [hunk 3]
```diff
@@ -23,17 +23,38 @@ Datadog is able to indicate the filename and line number where the vulnerability
...
 - Cookie without HttpOnly Flag
 - Cookie without SameSite Flag
```
Comment (urseberry) [inline content/en/security/application_security/code_security/_index.md:40]:
```suggestion
- Cookie without HttpOnly flag
- Cookie without SameSite flag
```

File: content/en/security/application_security/code_security/_index.md [hunk 4]
```diff
@@ -23,17 +23,38 @@ Datadog is able to indicate the filename and line number where the vulnerability
...
+- Path traversal
+- Reflection injection
+- Server Side Request Forgery (SSRF)
+- Session timeout
+- Session Rewriting
```
Comment (urseberry) [inline content/en/security/application_security/code_security/_index.md:45]:
```suggestion
- Session rewriting
```

File: content/en/security/application_security/code_security/_index.md [hunk 5]
```diff
@@ -23,17 +23,38 @@ Datadog is able to indicate the filename and line number where the vulnerability
...
+- SQL injection
+- Stacktrace leak
+- Trust Boundary Violation
```
Comment (urseberry) [inline content/en/security/application_security/code_security/_index.md:48]:
```suggestion
- Stack trace leak
- Trust boundary violation
```

File: content/en/security/application_security/code_security/_index.md [hunk 6]
```diff
@@ -23,17 +23,38 @@ Datadog is able to indicate the filename and line number where the vulnerability
...
 - Unvalidated Redirect
+- Verb tampering
+- Weak Cipher
+- Weak Hash
+- Weak Randomness
+- X-Content-Type-Options Header missing
+- X-XSS-Protection Header disabled
+- XPath Injection
```
Comment (urseberry) [inline content/en/security/application_security/code_security/_index.md:56]:
```suggestion
- Weak cipher
- Weak hash
- Weak randomness
- X-Content-Type-Options header missing
- X-XSS-Protection header disabled
- XPath injection
```

**Review by urseberry — COMMENTED** [review]
Capitalization notes

**Review by urseberry — APPROVED** [review]

**Comment by urseberry** [comment]
> I don't understand why the filename is marked as invalid as I only edited the content of this file

This was due to a bug in our tooling that has since been fixed.

**Pull request merged**